backend/tests/unit/config/test_agent_config.py

```
@@ -14,7 +14,7 @@
 from tarsy.config.agent_config import ConfigurationLoader
 from tarsy.config.exceptions import ConfigurationError
 from tarsy.models.agent_config import CombinedConfigModel, AgentConfigModel, MCPServerConfigModel, ChainConfigModel, ChainStageConfigModel
-from tarsy.agents.constants import IterationStrategy
+from tarsy.models.constants import IterationStrategy
```

Ruff (0.12.2)

`tarsy.models.constants.IterationStrategy` imported but unused

Remove unused import: `tarsy.models.constants.IterationStrategy`

(F401)